web/components/space-homepage/SpaceLandingPage.tsx

```diff
@@ -110,16 +116,81 @@ export function SpaceLandingPage() {
     [profileListingData?.profile_listing]
   );
 
+  
   const filteredProfileListings = useMemo(() => {
     if (searchQuery === "")
       return shuffleProfiles(allProfileListings, getDayOfYear(new Date()));
 
     const searchQueryLower = searchQuery.toLowerCase();
     const fuse = new Fuse(allProfileListings, FUSE_OPTIONS);
-
     return fuse.search(searchQueryLower).map((result) => result.item);
   }, [allProfileListings, searchQuery]);
 
+  // loop through filteredProfileListings!
+  
+  // idToProfileScore (user's id : how "filled out" their profile is!)
+  const ids = [];
+  const profileScores = [];
+  const idsToProfileScores = new Map();
+
+  const { userData } = useUserData();
+  const [{ data: profileData }] = useAllProfilesOfUserQuery({
+    variables: { user_id: userData?.id ?? "" },
+  });
+  
+  
+  
+
+  for(let i = 0; i < filteredProfileListings.length; i++){
+    idsToProfileScores.set(filteredProfileListings[i].id, 0);
```

Comment:
Did you mean to add code to calculate the scores? Seems like all the scores are 0 and this variable remains unused.